libfwupdplugin/fu-usb-device-ms-ds20.c

```diff
@@ -18,7 +18,7 @@ G_DEFINE_TYPE(FuUsbDeviceMsDs20, fu_usb_device_ms_ds20, FU_TYPE_USB_DEVICE_DS20)
 #define USB_OS_20_SET_HEADER_DESCRIPTOR	      0x00
 #define USB_OS_20_SUBSET_HEADER_CONFIGURATION 0x01
 #define USB_OS_20_SUBSET_HEADER_FUNCTION      0x02
-#define USB_OS_20_FEATURE_COMPATBLE_ID	      0x03
+#define USB_OS_20_FEATURE_COMPATIBLE_ID	      0x03
```

Comment:
I've generally tried to retain whitespace alignment.

Some projects don't like changing defines, but this is a `.c` so it should be a local and harmless to rename.

Reply:
Just do what clang-format wants please